hack/update-bindata.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+if [ -z ${KUBE_ROOT} ]; then
+	echo "KUBE_ROOT not detected, setting default."
+	KUBE_ROOT="../../../"
+fi
+
+if [[ ! -d "${KUBE_ROOT}/examples" ]]; then
+	echo "${KUBE_ROOT}/examples not detected.  This script should be run from a location where the source dirs are available."
+	exit 1
+fi
+
+# Setup bindata if not already in the system.
+# For separation of concerns, download first, then install later.
+
+git config --global user.email "bindata-mockuser@example.com" && git config --global user.name "bindata-mockuser"
```

Comment:
well this is new and exciting.

I'm a little hesitant to have this here - someone might run `hack/update-bindata.sh` directly and inadvertently overwrite their local settings.

The issue is probably in the cross-build docker image (though I'm curious why we never hit this before). cc @zmerlynn @spxtr